app/config.py

```diff
@@ -0,0 +1,27 @@
+db_user = 'postgres'
+user_password = 'postgres'
+db_name = 'mydiary'
+
+class Config:
+    SECRET_KEY = "TcQsWISFjRG4243XobHPIaDxMioisOba"
+    DEBUG = False
+    TESTING = False 
+    DEVELOPMENT = False
+
+class DevelopmentConfig(Config):
+    DEBUG = True
+    DEVELOPMENT = True
+    DATABASE_URL = 'postgresql://'+ db_user +': '+ user_password + '@localhost/'+ db_name +''
```

Comment:
Please read about environment variables, I would prefer this being set as an environment variable.